code/game/objects/items/melee/misc.dm

```diff
@@ -304,6 +304,17 @@
 	attack_verb_continuous = list("flogs", "whips", "lashes", "disciplines")
 	attack_verb_simple = list("flog", "whip", "lash", "discipline")
 	hitsound = 'sound/weapons/whip.ogg'
+	
+/obj/item/melee/curator_whip/dadbelt
+	name = "complaints department"
```

Comment:
```suggestion
	name = "Complaints Department"
```
That's the name of the belt.

Reply:
I didn't see anything else using caps so I didn't either, my first instinct was to caps it since it's a proper noun